### PR TITLE
given answer

### PR DESCRIPTION
**Pull request by linuscodes56**
given answer to your question in js file

### CODE REVIEW

File: js1 [hunk 1]
```diff
@@ -1,2 +1,2 @@
 Q1 - hello how are you?
-ans - 
+ans - hello i am sunil kumar from sunil14701 and i am here just for testing
```
Comment (Sunil-Kumar56) [inline js1:2]: good change but can you also add your homestate

Comment (linuscodes56) [reply js1:2]: yay !sure

Comment (linuscodes56) [reply js1:2]: yay !sure
but why

Comment (Sunil-Kumar56) [reply js1:2]: just for your conformation.

**Comment by Sunil-Kumar56** [comment]
sure

**Pull request merged**